From 0f05c8e0c4833ab02dbe26c590334949cd85b5f9 Mon Sep 17 00:00:00 2001
From: Douglas Anderson <dianders@chromium.org>
Date: Thu, 10 Aug 2017 09:04:54 -0700
Subject: [PATCH] CHROMIUM: arm64: dts: rockchip: Fix rk3399-gru-nefario
 backlight

The PWM IDs for EC backlights come from the EC source base.  Thus the
one true ID mapping for the EC PWMs on nefario can be found at:

https://chromium.googlesource.com/chromiumos/platform/ec/+/ec99f3913791/board/nefario/board.h#212

You can see that PWM_CH_DISPLIGHT ends up being defined as 1 so our
backlight PWM is ID 1.

BUG=b:63537905
TEST=Compile

Change-Id: I8e1f258fca79f0e3bfed3c79ee0672c8463a7d71
Signed-off-by: Douglas Anderson <dianders@chromium.org>
Reviewed-on: https://chromium-review.googlesource.com/610449
Reviewed-by: Alexandru M Stan <amstan@chromium.org>
Reviewed-by: Brian Norris <briannorris@chromium.org>
---
 arch/arm64/boot/dts/rockchip/rk3399-gru-nefario-r0.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-gru-nefario-r0.dts b/arch/arm64/boot/dts/rockchip/rk3399-gru-nefario-r0.dts
index 0fb2a898e6c5..8a98f5a4942d 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-gru-nefario-r0.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-gru-nefario-r0.dts
@@ -97,7 +97,7 @@
 
 	backlight: backlight {
 		compatible = "pwm-backlight";
-		pwms = <&cros_ec_pwm 0>;
+		pwms = <&cros_ec_pwm 1>;
 		brightness-levels-scale = <0x10000>;
 		default-brightness-level = <0x828f>; /* arbitrary, ~51% */
 		enable-gpios = <&gpio1 17 GPIO_ACTIVE_HIGH>;
-- 
2.20.1

